src/appresolver.js

```diff
@@ -42,6 +42,7 @@ function LocalAppRegistry(appClass, opts) {
 	opts.port = opts.port || 3000;
 	opts.dist = opts.dist || 'dist';
 	opts.configFile = opts.configFile || 'appconfig.json';
+	opts.baseRoute = opts.baseRoute !== undefined ? opts.baseRoute : '/app';
```

Comment:
Can't use `||` operator here since `baseRoute = ''` will trigger the false condition and set it to `/app`. Tested this locally.